homeassistant/util/ruamel_yaml.py

```diff
@@ -0,0 +1,124 @@
+"""ruamel.yaml utility functions."""
+import logging
+import os
+from os import O_CREAT, O_TRUNC, O_WRONLY
+from stat import ST_MODE, ST_UID, ST_GID
+from collections import OrderedDict
+from typing import Union, List, Dict
+
+from ruamel.yaml import YAML
+from ruamel.yaml.constructor import SafeConstructor
+from ruamel.yaml.error import YAMLError
+from ruamel.yaml.compat import StringIO
+
+from homeassistant.util.yaml import _secret_yaml
```

Comment:
If we're using it in two files, it can't have a `_` prefix, that means it's private to that module.